libs/api/domains/endorsement-system/src/lib/dto/pagination.input.ts

```diff
@@ -0,0 +1,25 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString, IsNumber, IsOptional } from 'class-validator'
+
+@InputType()
+export class PaginationInput {
```

Comment:
This name is too broad and will most likely clash with other input that have the same name.
In order to fulfil the graphql naming conventions for island.is, this needs to be narrowed down to the project, e.g.
```
EndorsementPaginationInput
```

reference: https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#integrating-these-naming-conventions-into-our-shared-api